packages/ui/src/composables/useTranslation.ts

```diff
@@ -21,17 +35,17 @@ export const useTranslation = () => {
 
   return {
```

Comment:
Can we add here getCurrentInstance, find $i18n and try to $t. User may have i18n module installed and we can find value from their config, instead of our. I'd say, i18n confirm have priority over our translations config. But, I think we also need require user to prefix translations for vuestic. So it is `va:cancel`, not just `cancel`. We don't react if user changed language with i18n config - that's the main problem.

Reply:
Found it here: https://github.com/epicmaxco/vuestic-ui/pull/4256/files